### ISSUE TITLE
Where are the PL/SQL communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/plsql#37]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is PL/SQL supported by one or more large organizations?
* Does PL/SQL have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about PL/SQL? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for PL/SQL? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in PL/SQL? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in PL/SQL?
* Are there popular newsletters for PL/SQL?
* Is PL/SQL taught at programming bootcamps? (If so, what are some examples?)
* Is PL/SQL taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about PL/SQL?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by workingjubilee** [comment]
Oracle, obviously.

However, there is also a mildly divergent but highly effective implementation supported by PostgreSQL (called PL/pgSQL) and MariaDB (called Stored Procedures, but they also offer an Oracle compatibility mode), which are open source. Given that association with Oracle seems to be an active turnoff for engagement, according to [this remark](https://github.com/exercism/plsql/issues/51#issuecomment-542882194), it might be wise to inquire with one of those organizations about structuring an SQL track around their offerings. 

PL/SQL -> PL/pgSQL seems like it would offer the least friction, and especially be fruitful, given that it is a highly popular open source database. However, there are still other implementations of what might better be called [PL/SQM](https://en.wikipedia.org/wiki/SQL/PSM), which is a standard based on Oracle's PL/SQL. Obviously, studying PL/SQL seems like it would give one a leg up with any SQL/PSM implementation, but new students may not understand that at all.

**Comment by dgdosen** [comment]
I would be more open to looking at this track if it was based on open sql - like postgres.  

For one, I have postgres and related tools 'handy'.  That's not the case for the oracle environment.
